lib/active_model/serializer.rb

```diff
@@ -83,37 +79,6 @@ def self.type(type)
       self._type = type
     end
 
-    # @example
-    #   class AdminAuthorSerializer < ActiveModel::Serializer
-    #     attributes :id, :name, :recent_edits
-    def self.attributes(*attrs)
-      attrs = attrs.first if attrs.first.class == Array
-
-      attrs.each do |attr|
-        attribute(attr)
-      end
-    end
```

Comment:
wrote a bunch but github seems to have lost it when I clicked on the tab to get the link to https://github.com/rails-api/active_model_serializers/pull/1262#issuecomment-151545353 re: a latent Attribute class and not removing the attributes api definition from the serializer or changing the behavior of _reflections or _attributes or _attribute_keys 

gonna tweet about it.

tl;dr I'd rather discuss adding a new superclass (the Attributes mxin) separately from adding new behavior (removing the object instance method in favor of defining attribute and association mappings in the attribute / association declarations)